### PR TITLE
{data}[intel/2019b] Arrow v0.16.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3128.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/3fe86c64108b04b7d38895930bc250c6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2568.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/99b4d7d1e12bbaa07f3826e804ff4e2d for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/d3db37d38620b76b62a3061e16a72467 for a full test report.

File: easybuild/easyconfigs/a/Arrow/Arrow-0.16.0_fix-intel.patch [hunk 1]
```diff
@@ -0,0 +1,74 @@
+- avoid "Unknown compiler" CMake error
+- fix compilation error by replacing '__force_inline' (no longer known by Intel compiler) with 'inline'
```
Comment (zao) [inline easybuild/easyconfigs/a/Arrow/Arrow-0.16.0_fix-intel.patch]: Github diff viewer renders this list as if they were removals. I guess this is only cosmetic as they're not in a `---/+++` section?

Comment (boegel) [reply easybuild/easyconfigs/a/Arrow/Arrow-0.16.0_fix-intel.patch]: ugh, ok, will change

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/01419cf4dab1598e8d553fa3940257cb for a full test report.

**Comment by zao** [comment]
Going in, thanks @boegel!

**Pull request merged**